amd64/root/start.sh

```diff
@@ -40,6 +49,14 @@ if [ "$DECONZ_VNC_MODE" != 0 ]; then
   
   # Export VNC display variable
   export DISPLAY=$DECONZ_VNC_DISPLAY
+
+  # Generate SSL certificate
```

Comment:
What about an option `DECONZ_NOVNC_PORT=0` to skip this part?

Reply:
As this one gets `EXPOSE`d switch could be handled by an extra option or with a special value for `DECONZ_VNC_MODE`.

Reply:
Will be resolved, this is a good recommendation.